Sources/Orbit/Components/Badge.swift

```diff
@@ -22,64 +24,68 @@ public struct Badge<LeadingIcon: View, TrailingIcon: View>: View, PotentiallyEmp
                 leadingIcon
                     .iconSize(.small)
                     .font(.system(size: Icon.Size.small.value))
-                    .foregroundColor(labelColor)
+                    .foregroundColor(resolvedLabelColor)
 
                 Text(label)
                     .textSize(.small)
                     .fontWeight(.medium)
-                    .textLinkColor(.custom(labelColor))
+                    .textLinkColor(.custom(resolvedLabelColor))
                     .frame(minWidth: minTextWidth)
 
                 trailingIcon
                     .iconSize(.small)
                     .font(.system(size: Icon.Size.small.value))
-                    .foregroundColor(labelColor)
+                    .foregroundColor(resolvedLabelColor)
             }
-            .textColor(labelColor)
+            .textColor(resolvedLabelColor)
             .padding(.vertical, .xxSmall) // = 24 height @ normal size
             .padding(.horizontal, .xSmall)
             .background(
-                background
-                    .clipShape(shape)
+                resolvedBackground
+                    .clipShape(Capsule())
             )
         }
     }
 
-    @ViewBuilder var background: some View {
-        switch type {
-            case .light:                                Color.whiteDarker
-            case .lightInverted:                        Color.inkDark
-            case .neutral:                              Color.cloudLight
-            case .status(let status, true):             (status ?? defaultStatus).color
-            case .status(let status, false):            (status ?? defaultStatus).lightColor
-            case .custom(_, _, let backgroundColor):    backgroundColor
-            case .gradient(let gradient):               gradient.background
+    @ViewBuilder var resolvedBackground: some View {
+        if let backgroundColor = backgroundColor {
```

Comment:
We should really start using `SwiftFormat` in this project 🤔

Reply:
✔️